.github/workflows/test-signer-linux.yml

```
@@ -13,6 +13,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Bootstrap SoftHSM Credentials
+    - run: ./scripts/sofths_setup.sh
```

```suggestion
    run: ./scripts/sofths_setup.sh
```

Looks like this change accidentally sliced of the rest of the workflow?